kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/Kura8021xEAP.java

```diff
@@ -0,0 +1,7 @@
+package org.eclipse.kura.nm;
+
+public enum Kura8021xEAP {
+    tls,
+    peap,
+    ttls,
```

Comment:
Please use all caps enums to be consistent with the already established convention and to make Sonar happy.

Refer to https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/KuraIp6AddressGenerationMode.java and https://github.com/eclipse/kura/blob/develop/kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/KuraIp6Privacy.java as examples